halibot/halauth.py

```diff
@@ -47,20 +57,24 @@ def write_perms(self):
 
 	def grantPermission(self, ri, identity, perm):
 		if not self.enabled:
-			return
+			return False
 
 		t = (ri, identity, perm)
 		if t not in self.perms:
 			self.perms.append(t)
+			return True
+		return False
```

Comment:
NB: Should we raise an exception here instead of returning false?

Reply:
Probably. I have a lot of return bools in here, most of those should probably be exceptions. Too much C